lib/superstore/associations.rb

```diff
@@ -20,25 +20,25 @@ module ClassOverrides
       #   end
       def belongs_to(name, options = {})
         if options.delete(:superstore)
-          Superstore::Associations::Builder::BelongsTo.build(self, name, options)
+          Superstore::Associations::Builder::BelongsTo.build(self, name, **options)
         else
-          super
```

Comment:
is `super` deprecated?

Reply:
Nope. The problem here is we are making a super call to an active record method with a double splat param. 

https://github.com/rails/rails/blob/master/activerecord/lib/active_record/associations.rb#L1368

With the way we define our `belongs_to` method to take a plain Hash, Ruby 2.7 complains that `warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call`.

So this gives us two options: we can convert it back into keyword args on the way out or, as a found after a little more experimenting, the better approach which is switching our `options` param to be a double splat too, that way we can have plain calls to super, and we will pass what the ActiveRecord method is expecting. 

I'm going to switch to that fix instead.